cmake/macros/macro_expand_instantiations.cmake

```diff
@@ -78,7 +78,12 @@ MACRO(EXPAND_INSTANTIATIONS _target _inst_in_files)
   #
   FOREACH(_build ${DEAL_II_BUILD_TYPES})
     STRING(TOLOWER ${_build} _build_lowercase)
+
     ADD_DEPENDENCIES(${_target}.${_build_lowercase} ${_target}.inst)
```

Comment:
No.

CMake determines the implicit link dependencies during compile time. During configure time without looking at a file such a dependency cannot be constructed.